validations/src/validations.rs

```diff
@@ -261,28 +260,29 @@ pub fn validate_mint_transaction(
     let block_reward_value = block_reward(mint_tx.epoch);
     // Mint value must be equal to block_reward + transaction fees
     if mint_value != total_fees + block_reward_value {
-        Err(BlockError::MismatchedMintValue {
+        return Err(BlockError::MismatchedMintValue {
             mint_value,
             fees_value: total_fees,
             reward_value: block_reward_value,
         }
-        .into())
-    } else {
-        let outputs_len = mint_tx.outputs.len();
+        .into());
+    }
 
-        if !mint_tx.outputs.iter().map(|vto| vto.pkh).all_equal() {
-            return Err(BlockError::MultiplePkhsInMint.into());
-        }
+    if mint_tx.outputs.len() > 2 {
```

Comment:
I think this is not tested? That creating a mint with 2 outputs is fine but a mint with 3 outputs fails.

Reply:
line 149 of validations/src/tests.rs

Reply:
There is a misleading comment there:

    // Error: Mint outputs smaller than collateral minimum